shared-types/types.ts

```diff
@@ -15,10 +15,12 @@ export interface IUserTokens {
   refreshToken: string;
 }
 
+export type IDataSourceType = 'api' | 'tripadvisor' | 'google';
```

Comment:
maybe we can define an `enum` and create the type based on the `enum` so that we dont use plaintext values